pkg/kubectl/cmd/auth/cani.go

```diff
@@ -393,3 +395,13 @@ func isNamespaced(gvr schema.GroupVersionResource, discoveryClient discovery.Dis
 
 	return false, fmt.Errorf("the server doesn't have a resource type '%s' in group '%s'", gvr.Resource, gvr.Group)
 }
+
+func isVerb(s string) error {
+	verbs := []string{"create", "watch", "post", "put", "list", "get", "delete", "deletecollection", "update", "patch"}
+	for _, verb := range verbs {
+		if strings.ToLower(s) == verb {
```

Comment:
if we're trying to warn about possible typos, I wouldn't change case of the specified verb… a case-mismatch would affect authz checks